app/src/main/java/org/simple/clinic/summary/PatientSummaryInit.kt

```diff
@@ -36,4 +30,18 @@ class PatientSummaryInit : Init<PatientSummaryModel, PatientSummaryEffect> {
 
     return first(model, effects)
   }
+
+  private fun checkTeleconsultationInfo(
+      model: PatientSummaryModel,
+      effects: MutableSet<PatientSummaryEffect>
+  ) {
+    when (model.teleconsultInfo) {
+      null, is TeleconsultInfo.Fetching -> {
+        effects.add(FetchTeleconsultationInfo(model.currentFacility!!.uuid))
+      }
+      is TeleconsultInfo.NetworkError -> {
+        effects.add(ShowTeleconsultInfoError)
+      }
```

Comment:
What happens in the remaining cases of `TeleconsultInfo`?
Can we add an `else` here?

Reply:
We are not handling any other cases. So else will be empty, is that fine?

Reply:
Leave it be. We don't need to add `else`

Reply:
Already added it with a comment `// Do nothing` 😅